app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -103,6 +119,9 @@ class GithubWebhook extends RequestHandler<Body> {
             pr.head.sha,
             'Pull request closed',
           );
+        } else {
+          // Merged pull requests can be added to CI.
+          await scheduler.addPullRequest(pr);
```

Comment:
Should we be listening for merged PRs, or commits?

Reply:
I don't think there's a commit webhook event - https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads

The PR event provides a lot of data, and it provides the release branch info. I'll add a test to verify release PRs.

Reply:
Offline conversation. There is a commit webhook called push - https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push

It looks like the GitHub library doesn't support that event yet, so I'm going to look into adding it

Reply:
I looked into the push events on the archive table, and they're not the solution. We need to support this event + push events to get 100% coverage. From the last 90 days, push events made up 0% of commits that need to be tested by infra.

I'm going to follow up and modify the cron job from every minute to every hour, and have it check for any missing commits. Right now it only adds newer, missing commits.

Reply:
Filed https://github.com/flutter/flutter/issues/78351 for tracking the PushEvent addition

Reply:
We sync'ed up offline again, and triggering based on PR merged even SGTM